GiniVision/Classes/Extensions.swift

```diff
@@ -9,6 +9,7 @@
 import UIKit
 import AVFoundation
 import ImageIO
+import MobileCoreServices
```

Comment:
Why do you need that `MobileCoreServices `?

Reply:
Because of the _UTI's_